tools/src/android-update-native-dependencies/dependenciesChangelogs.json

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
We will need to find a better place for such jsons 🤔 We also have `changelogVersions.json` in the root folder (which is updated by `et merge-changelogs`) — might be good to keep them close — maybe `tools/data`?. What do you think?